backend/handlers/handlers.go

```diff
@@ -702,11 +702,16 @@ func (handlers *Handlers) postBtcFormatUnit(r *http.Request) interface{} {
 }
 
 // getAccountsBalanceHandler returns the balance of all the accounts, grouped by keystore and coin.
-func (handlers *Handlers) getAccountsBalance(*http.Request) (interface{}, error) {
+func (handlers *Handlers) getAccountsBalance(*http.Request) interface{} {
+	type response struct {
+		Success     bool                                                     `json:"success"`
+		ErrorCode   string                                                   `json:"errorCode,omitempty"`
+		TotalAmount map[string]map[coin.Code]accountHandlers.FormattedAmount `json:"formattedAmount,omitempty"`
+	}
```

Comment:
The frontend type does not match this:

```
export type TAccountsBalanceResponse = {
  success: true;
  balance: TAccountsBalance;

} | {
  success: false;
  errorMessage?: string;
  errorCode?: 'syncInProgress';
}

```

As mentioned before, you should set the ErrorCode only if it's an actual error code, otherwise put it into ErrorMessage.

Reply:
Fixed and removed the errocode in the api as discussed